intranet/static/js/common.header.js

```diff
@@ -57,7 +63,7 @@ $(function() {
             $(document).unbind("click tap", clickOutside);
             $(".dropdown-menu").unbind("click", clickInside);
             $(".has-dropdown").removeClass("dropdown-open");
-        } else if (already_open.length == 0) {
+        } else if (already_open.length === 0) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'already_open' is not in camel case.](https://www.codacy.com/app/tjhsst-ion/ion/file/2756143378/issues/source?bid=3212528&fileBranchId=3355681#l66)